CHANGELOG.md

```diff
@@ -56,7 +58,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - First release.
 
-[Unreleased]: https://github.com/giantswarm/helmclient/compare/v1.0.2...HEAD
+[Unreleased]: https://github.com/giantswarm/helmclient/compare/v1.0.4...HEAD
+[1.0.4]: https://github.com/giantswarm/helmclient/compare/v1.0.2...v1.0.4
```

Comment:
Not sure how much it matters, but I guess it should be like this:

```suggestion
[1.0.4]: https://github.com/giantswarm/helmclient/compare/v1.0.3...v1.0.4
```

Reply:
I found out v1.0.3 was not in changelog. I fixed it already.

Reply:
Great!